core/domain/image_services_test.py

```diff
@@ -66,25 +64,19 @@ def test_compress_image_returns_correct_dimensions(self):
     def test_invalid_scaling_factor_triggers_value_error(self):
         value_exception = self.assertRaisesRegexp(
             ValueError,
-            re.escape(
-                'Scaling factor should be in the interval (0, 1],'
-                ' received 1.1.'))
+            r'Scaling factor should be in the interval \(0, 1], received 1.1.')
```

Comment:
Why cannot we use escape here (applies for everything this file).

Reply:
Done.